src/log.js

```diff
@@ -540,25 +559,45 @@ export function resetLogConstructorForTesting() {
  *  2. Development mode is enabled via `#development=1` or logging is explicitly
  *     enabled via `#log=D` where D >= 1.
  *
+ * @param {!Element=} opt_element
  * @return {!Log}
  */
-export function user() {
+export function user(opt_element) {
+  let logger;
```

Comment:
@lannka i think it is because I'd need to assign different value to this var and also use it at the last return call. Lmk what you think!